build.sh

```diff
@@ -30,19 +24,12 @@ function build_grub() {
 }
 
 function build_qemu() {
-  # Dependencies
-  if [ -x "$(command -v apt)" ]; then
-    sudo apt -qq install git \
-                         zlib1g-dev \
-                         libglib2.0-dev libfdt-dev libpixman-1-dev \
-                         libsdl2-dev libvulkan-dev
-  fi
   # Building
   cd ${path_orbital}
   cd ${path_qemu}
   ./configure --target-list=ps4-softmmu \
     --enable-sdl --enable-vulkan --enable-debug --disable-capstone \
-    --enable-hax
+    --enable-hax --disable-stack-protector
```

Comment:
Sorry, but this is not a viable solution. We need to understand why this happens and if it's required under certain circumstances we should add this conditionally.